specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -1962,6 +2004,18 @@
         }
       }
     },
+    "ElasticVersionsListResponse": {
+      "description": "List of elastic versions available for the monitor resource",
+      "type": "object",
+      "properties": {
+        "versions": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
List response schema should have a"value" property and the actual array should be inside value

Reply:
along with a nextLink. Look at other examples in the same file

Reply:
At max, we will get 10 records. Not more than that.

Reply:
ok. please keep in mind that anytime in future if you would want to change this to a pageable result it will be flagged as a breaking change and will need you to get a breaking changes approval.